oidc_config.go

```diff
@@ -36,3 +36,14 @@ func (c *Config) DiscoverEndpoints() {
 	assignIfEmpty(&c.IntrospectionEndpoint, discoveryJson["introspection_endpoint"].(string))
 	assignIfEmpty(&c.UserinfoEndpoint, discoveryJson["userinfo_endpoint"].(string))
 }
+
+type CustomArgs []string
+
+func (c *CustomArgs) String() string {
```

Comment:
Is the idea these will be used later on?

Reply:
I figured it might be useful for other commands too?

Reply:
Oh you meant the Strings() method I guess. I got the impression it was required by the flags package, but didn't look further into it.

Reply:
I meant that this String() function is not used anywhere. Is it required by flags?

Reply:
Let's not worry about that for now, we can adjust validation and stuff later.